### PR TITLE
OWTF 1.0.1 Lion_Heart: Cleaning up files and adding debian scripts

### PR DESCRIPTION
**Pull request by C0smos**
Sorry about the large amount of times I've managed to screw up trying to issue this pull request for the Debian install scripts. Hopefully this time it's all correct. Apologies again for the mistakes.

### CODE REVIEW

**Comment by DePierre** [comment]
@C0smos Don't apologize that much, that's no big deal here ;) @all Anyone with Debian between their hands to test?

**Comment by delta24** [comment]
Hi @C0smos, can you update your pull request to also include a script to create a file, named "/etc/apt/preferences.d/my_pref" or with any apt name, with this content

```
Package: *
Pin: release a=stable
Pin-Priority: 1000

# The following is if youare also using unstable and/or testing repo
#Package: *
#Pin: release a=testing
#Pin-Priority: 950

#Package: *
#Pin: release a=unstable
#Pin-Priority: 900

Package: *
Pin: release n=kali
Pin-Priority: 850

Package: *
Pin: release n=kali-bleeding-edge
Pin-Priority: 800
```
This will keep Debian as “clean” as possible by prioritizing the installation source, since both the sources have common packages.

**Comment by C0smos** [comment]
Hey @delta24 I've just added the script for the package preferences. 

Thanks

**Comment by delta24** [comment]
@C0smos Thanks and can you open a PR for the develop branch instead of master?

File: scripts/db_run.sh [hunk 1]
```diff
@@ -30,9 +30,9 @@
 # extensively on Kali
 
 get_config_value(){
-
+    current_user=$(logname)
     parameter=$1
-    file=$2
+    file="/home/$current_user/.owtf/db.cfg"
```
Comment (delta24) [inline scripts/db_run.sh]: This can be simplified by using just `$HOME/.owtf/db.cfg`.

File: install/install.py [hunk 2]
```diff
@@ -62,8 +62,11 @@ def install_in_directory(self, directory, command):
             print("[!] Directory %s already exists, so skipping installation for this"%(directory))
 
     def install_using_pip(self, requirements_file):
-        # Instead of using file directly with pip which can crash because of single library
-        self.run_command("sudo -E pip2 install --upgrade -r %s"%(requirements_file))
+         #Instead of using file directly with pip which can crash because of single library
+    	 if self.distro_num == 3:
```
Comment (delta24) [inline install/install.py]: Why this change? No difference here when using `sudo -H` or `sudo -E`.

File: install/install.py [hunk 3]
```diff
@@ -85,8 +88,8 @@ def install(self):
             print("(0) %s"%("My distro is not listed :("))
             distro_num = raw_input("Select a number based on your distribution : ")
             try: # Cheking if valid input is received
-                distro_num = int(distro_num)
-                if distro_num != 0:
+                self.distro_num = int(distro_num)
```
Comment (delta24) [inline install/install.py]: Same here.

File: scripts/db_setup.sh [hunk 4]
```diff
@@ -31,9 +31,10 @@
 
 get_config_value(){
 
+    current_user=$(logname)
     parameter=$1
-    file=$2
-
+    file="/home/$current_user/.owtf/db.cfg"
```
Comment (delta24) [inline scripts/db_setup.sh]: Same here. Not necessary.

**Comment by C0smos** [comment]
I've just opened a PR for the develop branch. Regarding the sudo issue, I can't remember exactly why I did that, as you said it's makes no difference :). 

Let me know if there's anything else
